### ISSUE TITLE
How to make transfer learned model only predict my own labels

### ISSUE THREAD

**Issue by Magsun** [open-mmlab/mmdetection#1987]
您好，我在试着用mmdetection框架中的一个预训练模型来迁移学习自己的类别，在训练的时候我修改了训练的配置，能够正常训练，但是在预测的时候偶尔模型会预测出不在训练类别中的标签。经过查看coco数据集的类别，推测可能是模型输出了coco预训练过的类别。
请问在哪里修改配置让模型只输出预训练过的类别？
非常感谢！

**Comment by ZwwWayne** [comment]
Please use English or English&Chinese for issues.

**Comment by Magsun** [comment]
> Please use English or English&Chinese for issues.

Sorry for that. Here's my question:

I'm trying to use pretrained cascade maskrcnn model transfer learning on my own dataset.

The dataset has only 3 categories, and the training process is ok.

But when I eval this new model on test dataset, it gives labels not included on train/eval labels.

How can I do to let transfer learning model only predict trained labels?

Thanks a lot.

**Comment by ZwwWayne** [comment]
You might need to check whether the `num_classes` is changed to be 3 (used for your own dataset) when you finetuning the pre-trained COCO model.

**Comment by CRmost** [comment]
You have better to create your own dataset to train the model.

**Comment by ChunmianLin** [comment]
@Magsun Hi, how to transfer the pretrained model to finetune on your own dataset? I also want to use the pretrained model, e.g. COCO pretrained model with 80 classes, to retrained on my own data (14 classes). But it reports the dimension error and I do not know how to solve it?
Could you please give me a hand?
